Cargo.toml

```diff
@@ -9,7 +9,7 @@ documentation = "https://docs.rs/bellperson"
 homepage = "https://github.com/filecoin-project/bellman"
 license = "MIT/Apache-2.0"
 repository = "https://github.com/filecoin-project/bellman"
-version = "0.12.2"
+version = "0.12.3"
```

Comment:
Note, I'm not manually advancing the version here -- but using 0.12.2 causes errors